rllib/core/models/configs.py

```diff
@@ -159,11 +159,11 @@ def build(self, framework: str = "torch") -> Model:
         self._validate(framework=framework)
 
         if framework == "torch":
-            from ray.rllib.core.models.torch.mlp import TorchMLPHead
+            from ray.rllib.core.models.torch.heads import TorchMLPHead
```

Comment:
Created a `heads` file (instead of mlp.py). Analogous to the already existing `encoders` file.